pkg/utils/configMapUtils.go

```diff
@@ -30,14 +31,40 @@ func (clientSets ClientSets) ValidateConfigMap(configMapName string, experiment
 }
 
 // SetConfigMaps sets the value of configMaps in Experiment Structure
-func (expDetails *ExperimentDetails) SetConfigMaps(clients ClientSets) {
+func (expDetails *ExperimentDetails) SetConfigMaps(clients ClientSets, engineName string) {
```

Comment:
Let's return the error from this function also and instead of printing error in this function you can print in the parent function.